lib/appsignal_phoenix/live_view.ex

```diff
@@ -86,21 +106,42 @@ defmodule Appsignal.Phoenix.LiveView do
     end)
   end
 
-  def handle_event_start(
-        [:phoenix, _type, name, :start],
+  def handle_live_view_event_start(
+        [:phoenix, :live_view, name, :start],
         %{system_time: system_time},
         metadata,
         _event_name
       ) do
     "live_view"
     |> @tracer.create_span(nil, start_time: system_time)
-    |> @span.set_name("#{Appsignal.Utils.module_name(metadata[:socket].view)}##{name}")
+    |> @span.set_name(span_name(metadata[:socket].view, name, metadata[:event]))
     |> @span.set_attribute("appsignal:category", "#{name}.live_view")
     |> @span.set_attribute("event", metadata[:event])
     |> @span.set_sample_data("params", metadata[:params])
     |> @span.set_sample_data("session_data", metadata[:session])
   end
 
+  def handle_live_component_event_start(
+        [:phoenix, :live_component, name, :start],
+        %{system_time: system_time},
+        metadata,
+        _event_name
+      ) do
+    span =
+      "live_component"
+      |> @tracer.create_span(nil, start_time: system_time)
+      |> @span.set_name(span_name(metadata[:component], name, metadata[:event]))
+      |> @span.set_attribute("appsignal:category", "#{name}.live_component")
+      |> @span.set_attribute("event", metadata[:event])
+      |> @span.set_sample_data("params", metadata[:params])
```

Comment:
Is there a reason why we cannot keep the `live_view` namespace here?

I think it does make sense to have a single namespace for both `LiveView` and `LiveComponent` instrumentation. This would allow to keep metrics such as throughput etc. in a single namespace.

Reply:
Ah, that makes a lot of sense. I have noticed that convention used widely. I'll modify the changes in this PR to use a single `live_view` namespace for both LiveView and LiveComponent actions.

Thank you again for the feedback!